From ac3225bd5a53cd6eb934ff96f537b4cf0334a2bd Mon Sep 17 00:00:00 2001
From: html6405 <peter.schelchshorn@mhs-solutions.at>
Date: Wed, 16 Feb 2022 15:56:00 +0100
Subject: [PATCH 13/14] Revert "SubscriptionInfoUpdater java  trigger
 radioreset when the sim becomes unknown"

This reverts commit 9bf199d9

Change-Id: Ia1881834ab07d6ae5e209bf312158eed694107e2
---
 .../com/android/internal/telephony/SubscriptionInfoUpdater.java | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/java/com/android/internal/telephony/SubscriptionInfoUpdater.java b/src/java/com/android/internal/telephony/SubscriptionInfoUpdater.java
index b2cca37d48..2b53bde031 100644
--- a/src/java/com/android/internal/telephony/SubscriptionInfoUpdater.java
+++ b/src/java/com/android/internal/telephony/SubscriptionInfoUpdater.java
@@ -281,8 +281,6 @@ public class SubscriptionInfoUpdater extends Handler {
                 broadcastSimApplicationStateChanged(msg.arg1, TelephonyManager.SIM_STATE_UNKNOWN);
                 updateSubscriptionCarrierId(msg.arg1, IccCardConstants.INTENT_VALUE_ICC_UNKNOWN);
                 updateCarrierServices(msg.arg1, IccCardConstants.INTENT_VALUE_ICC_UNKNOWN);
-                if (RIL.needsOldRilFeature("fakeiccid"))
-                    SystemProperties.set("gsm.radioreset", "true");
                 break;
 
             case EVENT_SIM_IO_ERROR:
-- 
2.37.1

